package.json

```diff
@@ -31,9 +32,11 @@
     "test": "npm run build && mocha test",
     "test:debug": "npm run build && mocha debug test",
     "coverage": "npm run build && c8 mocha && c8 report --reporter=html && echo Coverage report is available at ./coverage/index.html",
-    "prepublishOnly": "npm run test"
+    "prepublishOnly": "npm run test",
+    "types": "tsc -p ."
```

Comment:
Generating the types by running `npm run types` should automatically be done before publishing. Can you rename the script `build:types` and add running it to `prepublishOnly`?

Reply:
Or we can extend the `build` script to also generate the types, we have to test them afterwards too.